source/manual/manage-opensearch-on-aws.html.md

```diff
@@ -0,0 +1,65 @@
+---
+owner_slack: "#govuk-2ndline-tech"
+title: Manage OpenSearch on AWS
+section: Infrastructure
+layout: manual_layout
+parent: "/manual.html"
+---
+
+[AWS OpenSearch] is an open source, distributed search and analytics suite derived from Elasticsearch.
+
+## Access OpenSearch Dashboard
+
+We typically can't access an OpenSearch dashboard with the URL provided in the AWS console because the endpoint is in a private subnet in a [virtual private cloud (VPC)](https://en.wikipedia.org/wiki/Virtual_private_cloud). Therefore to access it we need to configure our system to tunnel into the subnet in the VPC.
+
+## Prerequisites
```

Comment:
```suggestion
### Prerequisites
```

Sorry, I should have added in my last suggestion that this header and the next one will need indenting.